docusaurus/website/pages/en/index.js

```diff
@@ -119,6 +128,110 @@ const Features = () => (
   </Block>
 );
 
+const GetStarted = props => (
+  <Block layout="twoColumn" background="light" {...props}>
+    {[
+      {
+        title: 'Get started coding in a matter of seconds!',
+        content: `With Create React App, you get to focus on **writing React, not boilerplate**.
+          All you need to do is run a command, install some dependencies, and decide what's for dinner.
+
+          npx create-react-app my-app
+          `,
+      },
+      {
+        image:
+          'https://camo.githubusercontent.com/29765c4a32f03bd01d44edef1cd674225e3c906b/68747470733a2f2f63646e2e7261776769742e636f6d2f66616365626f6f6b2f6372656174652d72656163742d6170702f323762343261632f73637265656e636173742e737667',
+        imageAlign: 'right',
+      },
+    ]}
+  </Block>
+);
+
+const Update = props => (
+  <Block layout="twoColumn" {...props}>
+    {[
+      {
+        image: imgUrl('update.png'),
+        imageAlign: 'left',
+      },
+      {
+        title: 'Easy-to-maintain toolchain',
+        content: `Keeping a build toolchain up to date with the latest and greatest can be a daunting and time-consuming
+        task for even the most seasoned developer. Create React App extracts all of those concerns into a single
+        dependency, which are **easy to update** and **battle tested by thousands**
+
+        npm install react-scripts@latest`,
+      },
+    ]}
+  </Block>
+);
+
+const FineGrainedFeatures = props => (
+  <Block layout="fourColumn" align="center" padding={['bottom']} {...props}>
+    {[
+      {
+        title: 'Webpack 4',
+        content:
+          'Webpack 4 gives you lightning fast rebuilds and code-splitting out of the box',
```

Comment:
Consider punctuation (e.g. add periods) throughout to be consistent?

Reply:
I landed on removing the punctuation that was there, as i thought it looked better with no punctuation than a period after a ton of one-sentence paragraphs. Totally agree that it should be consistent!

Reply:
Thanks

I guess not sure since some have two sentences? If you prefer no periods, prob better to just leave it as it was... periods for the two sentences, no punctuation for the others...